server/docs/api.md

```diff
@@ -28,6 +29,14 @@ $ curl -H "Accept: application/json" https://bookworms-api.vercel.app/api/ping
 
 ### GET /api/reviews
 
+
+This api does three things
+
+- Get all reviews (By default in descending order according to likes)
+- Get all reviews with a partular keyword in books name(Search)
+- Get reviews in ascending order of likes
```

Comment:
This API should be able to sort by `recency` as well: https://github.com/codeday-labs/bookworms/projects/1#card-64509875

Reply:
No worries, as long as you are clear now.